run.py

```diff
@@ -0,0 +1,175 @@
+#!/usr/bin/env python
+
+# Helper script to create recipes for CRAN R packages to submit to conda-forge.
+#
+# Setup checklist:
+#
+# 1. Put the name of the package(s) in packages.txt
+# 2. Add your GitHub username to the list of maintainers in extra.yaml
+# 3. Run `python run.py` in this directory
+# 4. Move the recipe directory to staged-recipes/recipes
+#
+# Installation requirements:
+#
+# conda, conda-build 2, python 3
+
+import os
+import re
+import shutil
+import subprocess as sp
+import sys
+
+# Setup checks -----------------------------------------------------------------
+
+if sys.version_info.major == 2:
+    sys.stderr.write('You need to have python 3 installed to use the helper script\n')
+    sys.exit(1)
+
+if not shutil.which('conda'):
+    sys.stderr.write('You need to have conda installed to use the helper script\n')
+    sys.exit(1)
+
+if not shutil.which('conda-build'):
+    sys.stderr.write('You need to have conda-build installed to use the helper script\n')
+    sys.stderr.write('Run: conda install -c conda-forge conda-build\n')
+    sys.exit(1)
+
+if not os.path.isfile('packages.txt'):
+    sys.stderr.write('Unable to find the file packages.txt.\n')
+    sys.stderr.write('Please check that it exists and that you are executing the script\n')
+    sys.stderr.write('in the same working directory as packages.txt.\n')
+    sys.exit(1)
+
+if not os.path.isfile('extra.yaml'):
+    sys.stderr.write('Unable to find the file extra.yaml.\n')
+    sys.stderr.write('Please check that it exists and that you are executing the script\n')
+    sys.stderr.write('in the same working directory as extra.yaml.\n')
+    sys.exit(1)
+
+# Process packages -------------------------------------------------------------
+
+with open('packages.txt', 'r') as f:
+   packages = f.readlines()
+   packages = [x.strip() for x in packages]
+
+for fn in packages:
+
+    if fn == '':
```

Comment:
`if not fn.strip()`

Reply:
I use `strip()` above, but I can change this to `if not fn:`